numpy/f2py/__init__.py

```diff
@@ -113,6 +112,33 @@ def compile(source,
             os.remove(fname)
     return status
 
-from numpy._pytesttester import PytestTester
-test = PytestTester(__name__)
-del PytestTester
```

Comment:
There's no need to move these lines - pytesttester is already designed to cost nothing on import

Reply:
It imports numpy, and the point is to avoid importing numpy during build type. See #17620

Reply:
Oh yeah, of course